src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupClient.java

```diff
@@ -1314,4 +1314,108 @@ public ElastigroupGetInstanceStatusResponse getInstanceStatus(String instanceId)
         return retVal;
     }
 
+    public Boolean startBeanstalkMaintenance(String groupId) {
+
+        Boolean retVal;
+
+        RepoGenericResponse<Boolean> startBeanstalkMaintenanceResponse = getSpotinstElastigroupRepo().startBeanstalkMaintenance(groupId, authToken, account);
+
+        if (startBeanstalkMaintenanceResponse.isRequestSucceed()) {
+            retVal = startBeanstalkMaintenanceResponse.getValue();
+        }
+        else {
+            List<HttpError> httpExceptions = startBeanstalkMaintenanceResponse.getHttpExceptions();
+            HttpError       httpException  = httpExceptions.get(0);
+            LOGGER.error(
+                    String.format("Error encountered while attempting to start beanstalk maintenance. Code: %s. Message: %s.",
+                            httpException.getCode(), httpException.getMessage()));
+            throw new SpotinstHttpException(httpException.getMessage());
+        }
+
```

Comment:
remove extra line space wherever is applied

Reply:
Done